keanu-python/tests/test_vertex.py

```diff
@@ -70,7 +70,7 @@ def test_vertex_can_observe_scalar(jvm_view):
     gaussian = Vertex(jvm_view.GaussianVertex, 0., 1.)
     gaussian.observe(4.)
 
-    assert type(gaussian.get_value()) == np.ndarray
+    assert isinstance(gaussian.get_value(), float)
```

Comment:
Here and everywhere else, probably better to check `type == float` cause that's more clear